src/components/seo.js

```diff
@@ -0,0 +1,88 @@
+/**
+ * SEO component that queries for data with
```

Comment:
action: consider copy directly the seo.js from the old repo. it includes more info